UKHO.PeriodicOutputService/UKHO.BESS.ConfigurationService/Services/IConfigurationService.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace UKHO.BESS.ConfigurationService.Services
+{
+    public interface IConfigurationService
+    {
+        void ProcessConfigs();
```

Comment:
Would a boolean return type be useful for this method?

Reply:
not at the moment.